CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+Hey, welcome to the party! 🎉
+
+Thank you so much to contribute to EasyMDE. 😘
```

Comment:
Not sure this is good English.
Maybe "Thank you so much for contributing to EasyMDE" ?

Reply:
Probably, I'm not a native english speaker.